README.rst

```diff
@@ -149,7 +149,7 @@ News
 
 - 2018/02/01: there is now some documentation about the architecture of
   triSYCL on GPU and accelerators with its device compiler based on
-  Clang/LLVM in `<doc/architecture.rst>`_. While this is wildly
+  Clang_/LLVM_ in `<doc/architecture.rst>`_. While this is wildly
   experimental, there is a growing interested around it and it is
```

Comment:
Not actually part of the commit, but worth editing probably: "While this is wildly experimental, there is a growing interested" -> "While this is wildly experimental, there is a growing interest"

Reply:
Good catch. I have updated the branch.